src/test/resources/features/signup.feature

```diff
@@ -0,0 +1,72 @@
+Feature: Sign Up
```

Comment:
Just a comment, we can talk about this later:
In this `.feature` file, you are using `Scenario Outline` with a single example. What do you think about this practice? Do you think it is more convenient than using just `Scenario`?

Reply:
ok :+1: